### PR TITLE
Design

### PR DESCRIPTION
**Pull request by obits13**
Alex, can you review the Python changes?
George, can you review the loading thing changes?

### CODE REVIEW

File: frontend/rldb/src/InstanceCard.js [hunk 1]
```diff
@@ -9,12 +9,17 @@ class InstanceCard extends Component {
...
+    getApiType (type) {
+        var apiTypes = {"crate" : "crates", "body":"bodies", "paint":"paints", "player":"players", "wheel":"wheels"};
+        return apiTypes[type];
+    }
 
     render() {
         return (
             <div className=" col-md-4 col-sm-6 text-center">
-                <a href="">
+                <a href={"/" + this.getApiType(this.props.data.type) + "/" + this.props.data.id}>
```
Comment (obits13) [inline frontend/rldb/src/InstanceCard.js:22]: We may or may not need this... but the URL sort of matches what the API wants.
ex. /crates/8892 => /api/crates/8892

**Review by azagon63 — APPROVED** [review]

**Comment by azagon63** [comment]
I think this works, we'll just need to map github ID's to names somewhere. The way before just pulled the name from the commit, but this only gets usernames. I tried looking up names based on username, but not everyone put their full name on their account.

**Review by hbkse — APPROVED** [review]
I'm not sure the loading animation works how we want it, but let's see what happens. My guess is that at times you'll see either an eternal loading screen or nothing at all.

**Comment by obits13** [comment]
When the state gets set (the data finishes fetching from the API), react calls render() on the component again, so the loading screen gets replaced with whatever we wanted originally.

I'm manually calling everyone's usernames for the commit data, so we don't have to worry about mapping for now.

**Pull request merged**